src/components/Posts/PostMenu.tsx

```diff
@@ -50,25 +50,25 @@ const PostMenu = ({ isExpandedPost = false, post }: PostMenuProps) => {
 
   return (
     <>
-      <IconButton ref={menuRef} onClick={() => setMenuOpen(true)}>
+      <IconButton onClick={() => setMenuOpen(true)} ref={menuRef}>
         <MoreVert />
       </IconButton>
       <Menu
         anchorEl={menuRef.current}
-        open={menuOpen}
+        MenuListProps={{ sx: styles.menuList }}
         onClose={() => setMenuOpen(false)}
+        open={menuOpen}
         PaperProps={{
```

Comment:
unrelated but with the recent upgrade for MUI, I think we should open another ticket to update some of the older implementations

Reply:
Yea you're right, I made a new ticket